README.md

```diff
@@ -246,7 +247,42 @@ Uncomment and edit the below line in the script to use Cmder config even when la
 # CMDER_ROOT=${USERPROFILE}/cmder  # This is not required if launched from Cmder.
 ```
 
-### Integrating Cmder with [Hyper](https://github.com/zeit/hyper), [Microsoft VS Code](https://code.visualstudio.com/), and your favorite IDEs 
+### Handling with custom arguments when using init.bat
+
+You can pass custom arguments to `init.bat` and use `%flag_exists%` to detect it.
+
+It is useful when you have multiple modes to execute cmder.
+
+If you use
+
+```batch
+
+init.bat /startNotepad
```

Comment:
Is using `init.bat /startNotepad` the only method, or does `Cmder.exe /startNotepad` also work?

Reply:
`Cmder.exe /startNotepad` **DO NOT WORK.**
See https://github.com/cmderdev/cmder/pull/1758#discussion_r217874087